.bulldozer.yml

```diff
@@ -53,14 +53,14 @@ merge:
 
 ## "update" defines how and when to update pull request branches. Unlike with
 ## merges, if this section is missing, bulldozer will not update any pull requests.
-#update:
+update:
 #  # "whitelist" defines the set of pull requests that should be updated by
 #  # bulldozer. It accepts the same keys as the whitelist in the "merge" block.
 #  whitelist:
```

Comment:
@schmmd shouldn't the elements `whitelist:` and `blacklist:` be uncommented as well? Otherwise, it shouldn't work well IMHO.

Reply:
Hmmm... not quite sure how I made that mistake... thanks.

Reply:
See https://github.com/allenai/allennlp/pull/3727